Documentation/CodeSnippets/Events/Core/Resource/AfterFileMetaDataUpdatedEvent.rst.txt

```diff
@@ -0,0 +1,29 @@
+.. =========================================================
+.. Automatically generated by the TYPO3 Screenshots project.
+.. https://github.com/TYPO3-Documentation/t3docs-screenshots
+.. =========================================================
+..
+.. Extracted from TYPO3\CMS\Core\Resource\Event\AfterFileMetaDataCreatedEvent
+
+.. php:namespace::  TYPO3\CMS\Core\Resource\Event
+
+.. php:class:: AfterFileMetaDataUpdatedEvent
+
+   This event is fired once metadata of a file was updated, 
+   in order to update custom metadata fields accordingly.
+   
+   .. php:method:: getFileUid()
+   
+      :returntype: int
+      
+   .. php:method:: getMetaDataUid()
+   
+      :returntype: int
+      
+   .. php:method:: getRecord()
+   
+      :returntype: array
+      
+   .. php:method:: setRecord(array record)
+   
+      :param array $record: the record
```

Comment:
```suggestion
```

This method does not exist:

- https://github.com/TYPO3/typo3/blob/10.4/typo3/sysext/core/Classes/Resource/Event/AfterFileMetaDataUpdatedEvent.php
- https://github.com/TYPO3/typo3/blob/11.5/typo3/sysext/core/Classes/Resource/Event/AfterFileMetaDataUpdatedEvent.php
- https://github.com/TYPO3/typo3/blob/84ceebc2fd40e72cb3062b93236e9cffffa26e56/typo3/sysext/core/Classes/Resource/Event/AfterFileMetaDataUpdatedEvent.php